SocialCareCaseViewerApi/V1/Controllers/SubmissionController.cs

```diff
@@ -0,0 +1,57 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using SocialCareCaseViewerApi.V1.Boundary.Requests;
+using SocialCareCaseViewerApi.V1.Boundary.Response;
+using SocialCareCaseViewerApi.V1.Exceptions;
+using SocialCareCaseViewerApi.V1.UseCase.Interfaces;
+
+namespace SocialCareCaseViewerApi.V1.Controllers
+{
+    [ApiController]
+    [Route("api/v1/submissions")]
+    [Produces("application/json")]
+    [ApiVersion("1.0")]
+    public class SubmissionController : BaseController
+    {
+        private readonly ISubmissionsUseCase _submissionsUseCase;
+
+        public SubmissionController(ISubmissionsUseCase submissionsUseCase)
+        {
+            _submissionsUseCase = submissionsUseCase;
+        }
```

Comment:
What do we think about `CaseSubmissionController` and `CaseSubmissionUseCase`? For me as someone with no context, submission is quite vague. We're using `CaseSubmissionResponse` etc., unless they'll be other types of "submissions"?

Reply:
The Submissions are for forms in general that can be filled out for a resident.

How would you feel about `FormSubmissionController` and `FormSubmissionUseCase`?

Reply:
I like those suggestions! 👍